src/getThreadList.js

```diff
@@ -4,23 +4,32 @@ var utils = require("../utils");
 var log = require("npmlog");
 
 module.exports = function(defaultFuncs, api, ctx) {
-  return function getThreadList(start, end, callback) {
+  return function getThreadList(start, end, isPending, callback) {
```

Comment:
Could you show me some tips on how to make `pending` optional and `callback` required at the same time? 
Maybe I can do some type check on `pending`?